File: sem 5/PHP/Lab Exercise-8 OPPS Programming/Qu1.php
<?php
class Employee
{
	private $name;
	private $id;
	private $salary;

	function setEmployee($name,$id,$salary)
	{
		$this->name=$name;
		$this->id=$id;
		$this->salary=$salary;
	}

	function getEmployee()
	{
		echo "-------------------------------<br>";
		echo "Name : $this->name<br>";
		echo "ID : $this->id<br>";
		echo "Salary : $this->salary<br>";
		echo "-------------------------------<br>";
	}
}

$e1=new Employee();
$e1->setEmployee("Sujal",1,50000);

$e2=new Employee();
$e2->setEmployee("Kirtan",2,40000);

$e3=new Employee();
$e3->setEmployee("Purav",3,30000);

$e1->getEmployee();
$e2->getEmployee();
$e3->getEmployee();
?>

File: sem 5/Python/UNIT-4/Qu1.py
class Doctor:
	def __init__(self,name,spec):
		self.name=name
		self.spec=spec

	def __str__(self):
		return f"Dr. {self.name} ({self.spec})"

class Patient:
	def __init__(self,name,age):
		self.name=name
		self.age=age

	def __str__(self):
		return f"{self.name}, Age: {self.age}"

class Appointments:
	def __init__(self):
		self.appointment=[]

	def schedule(self,doctor,patient,time):
		self.appointment.append((doctor,patient,time))#appointment add as tuple
		print("-----------------------------------------------------------")
		print(f"Appointment : {patient} with {doctor} at {time}")

	def daily(self):
		if not self.appointment:
			print("No Appointments")
			return

		for doctor,patient,time in self.appointment:
			print("-----------------------------------------------------------")
			print(f"Time : {time},Doctor : {doctor},Patient : {patient}")

d1=Doctor("Sujal","Neurologist")
d2=Doctor("Kirtan","Pathology")

p1=Patient("Nihir",25)
p2=Patient("Manan",20)
p3=Patient("Purav",22)

hospital=Appointments()

hospital.schedule(d1,p1,"9:00 AM")
hospital.schedule(d2,p2,"12:00 PM")
hospital.schedule(d1,p3,"5:00 PM")

hospital.daily()